CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# Contributuing
+
+Contributions to this project are welcome, whether minor updates to documentation or entirely new features.
+This document outlines the technical and procedural requirements for your work to be included.
+All contributions, in any form, must also confirm with the [code of conduct](CODE_OF_CONDUCT.md).
+
+> :warning: This document is a living document, and will adapt based on the project's needs.
+> If some part of this document creates a hindrance, raise an issue explaining the problem.
+
+## Scope: What Lives in this Repository?
+
+MewBot's core design includes the tenant of modularity.
+The core repo will only accept features and components that do not add additional dependencies.
+
+When building new components, whether they are input/output modules or utilities for use in behaviours,
+consider if they could be a stand-alone repository and package.
+The MewBot team is happy to help set up these repositories in the GitHub group.
```

Comment:
For now, this should be commented out -- we can deal with splitting up the repo once #13 is solved